src/methods/properties.js

```diff
@@ -160,7 +160,7 @@ function InternalUpdatedProperty(realm: Realm, O: ObjectValue, P: PropertyKeyVal
           generator.emitPropertyAssignment(O, P, descValue);
         }
       } else {
-        generator.emitDefineProperty(O, P, desc);
+        generator.emitDefineProperty(O, P, desc, true);
```

Comment:
Use "generator.emitDefineProperty(O, P, desc, /*isDescChanged*/true);" to increase readability.